src/types/onyx/PersonalDetails.ts

```diff
@@ -81,8 +81,10 @@ type PersonalDetails = OnyxCommon.OfflineFeedback<'avatar' | 'originalFileName'>
     status?: Status;
 };
 
-type PersonalDetailsList = Record<string, PersonalDetails | null>;
+type PersonalDetailsWithOfflineFeedback = OnyxCommon.OnyxItemWithOfflineFeedback<PersonalDetails, keyof PersonalDetails>;
```

Comment:
Same, before it was only `'avatar' | 'originalFileName'`. Not sure if we should allow all properties to be inside `pendingFields`.